specs/BrowserProcessExited.md

```diff
@@ -0,0 +1,186 @@
+# Background
+We have heard asks for a WebView2 API to easily track the `WebView2 Runtime`'s
+browser process exit. Manually waiting for the process to exit requires
+additional work on the host app, so we are proposing the `BrowserProcessExited`
+event. The `ProcessFailed` already lets app developers handle unexpected browser
+process exits for a WebView, this new API lets you listen to both expected and
+unexpected process termination from the `ICoreWebView2Environment` interface so
+you can, e.g., cleanup the user data folder when it's no longer in use. In this
+document we describe the new API. We'd appreciate your feedback.
+
+
+# Description
+The `BrowserProcessExited` event allows developers to subscribe event handlers
+to be run when the `WebView2 Runtime`'s browser process associated to a
+`CoreWebView2Environment` terminates. A key scenario is cleanup of the use data
+folder used by the `WebView2 Runtime`, which is locked while the runtime's
```

Comment:
What are the other scenarios? Is moving to an updated version of WebView2 a scenario? If so, please add a sample for it.

Reply:
Yes, mostly cleanup and updating the WebView2 version.

Reply:
I updated `Description` and sample C# code for the update scenario.